README.md

```diff
@@ -1,29 +1,35 @@
-# Smart Dashboard Project
+# SmartDashboard Project
 
 [![Build Status](https://travis-ci.org/wpilibsuite/SmartDashboard.svg?branch=master)](https://travis-ci.org/wpilibsuite/SmartDashboard)
 
-Welcome to the WPILib project. This repository contains the Smart Dashboard project.
+Welcome to the WPILib project. This repository contains the SmartDashboard project.
 
 - [WPILib Mission](#wpilib-mission)
-- [Building Smart Dashboard](#building-smart-dashboard)
-    - [Requirements](#requirements)
-    - [Running Smart Dashboard](#running)
+- [Getting SmartDashboard](#getting-smartdashboard)
+- [Building SmartDashboard](#building-smartdashboard)
 - [Contributing to WPILib](#contributing-to-wpilib)
 
 ## WPILib Mission
 
 The WPILib Mission is to enable FIRST teams to focus on writing game-specific software rather than on hardware details - "raise the floor, don't lower the ceiling". We try to enable teams with limited programming knowledge and/or mentor experience to do as much as possible, while not hampering the abilities of teams with more advanced programming capabilities. We support Kit of Parts control system components directly in the library. We also strive to keep parity between major features of each language (Java, C++, and NI's LabVIEW), so that teams aren't at a disadvantage for choosing a specific programming language. WPILib is an open-source project, licensed under the BSD 3-clause license. You can find a copy of the license [here](BSD_License_for_WPILib_code.txt).
 
-# Building Smart Dashboard
+# Getting SmartDashboard
 
-Building Smart Dashboard is very straightforward. Smart Dashboard uses Gradle to compile.
+The latest release build of SmartDashboard can be found on [WPI's server](http://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/wpilib/SmartDashboard/).
```

Comment:
@AustinShalit, your browse doesn't show full links at the bottom right? What are you running? And @nightpool, the idea with first.wpi.edu is that we _don't_ want to host stuff.